.idea/misc.xml

```diff
@@ -28,7 +28,10 @@
     <option name="priority" value="Medium" />
     <option name="excludeFilter" value="" />
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="false" assert-keyword="true" jdk-15="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="PDMPlugin">
```

Comment:
after removing and restarting intellij, this block re-appears ; i'd prefer to keep this to prevent conflicts in the future...